.editorconfig

```diff
@@ -13,3 +13,5 @@ trim_trailing_whitespace = false
 [{.travis.yml,npm-shrinkwrap.json,package.json}]
 indent_style = space
 indent_size = 2
+[Makefile]
+indent_style = tab
```

Comment:
I personally program most of the time in `vim`. And the worst thing you can have there is different intent style, than you have in your `.vimrc`.

In general speaking, you try to find a standard which says tabs or spaces. and also the intent size. Not only for you, for the whole company/project. And as long as you do not configure this in a central config file or a per file config (like possible in vim), you should stay with one type because otherwise you need to create a special config file, per project, per filetype which is more than ugly...